tools/cxbe/Exe.cpp

```diff
@@ -115,7 +116,46 @@ Exe::Exe(const char *x_szFilename)
                 goto cleanup;
             }
 
-            printf("OK %d\n", v);
+            // interpret long section names
+            if(m_SectionHeader[v].m_name[0] == '/')
```

Comment:
I used objcopy to rename a section I made in some inline asm

Reply:
I have to say I'm not a fan, I'd rather stay a mile away from intentionally breaking the PE spec (or giving the impression that we support doing so).

Reply:
The thing with `/` is documented in the spec on microsoft's page. While it says that long section names are not supported with executables, objcopy can still produce executables with long section names and they load and run just fine.

Reply:
Yes, it is documented as not being valid for executable images. While it may be possible to hack around the linker for that, it *is* breaking the spec.

Reply:
Also I added this because I actually *need* this to have a title image.
It would incorrectly add debug sections and the image sections as something like `/43` which ofc would not be picked up by the dashboard as an icon.

Reply:
1. How does the XDK do it?
2. Can't we tell cxbe to add or rename sections? (Not that I'd like the added complexity, but it's probably better than breaking the spec)